charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -105,6 +105,11 @@ spec:
           securityContext:
           {{- toYaml .Values.securityContext | nindent 12 }}
           args:
+            {{ if .Values.e2eTestEnable }}
+            - "-test.coverprofile=/workspace/data/e2e-profile.out"
+            - "__DEVEL__E2E"
+            - "-test.run=E2EMain"
+            {{ end }}
```

Comment:
we can write a script and use string replace for this work instead of using chart values.

Reply:
done!